assets/xml/objects/object_ms.xml

```diff
@@ -1,32 +1,32 @@
 ﻿<Root>
     <File Name="object_ms" Segment="6">
-        <Animation Name="object_ms_Anim_0005EC" Offset="0x5EC" />
-        <DList Name="object_ms_DL_0021E0" Offset="0x21E0" />
-        <DList Name="object_ms_DL_002300" Offset="0x2300" />
-        <DList Name="object_ms_DL_0028A0" Offset="0x28A0" />
-        <DList Name="object_ms_DL_002CC8" Offset="0x2CC8" />
-        <DList Name="object_ms_DL_002E58" Offset="0x2E58" />
-        <DList Name="object_ms_DL_002F90" Offset="0x2F90" />
-        <DList Name="object_ms_DL_0030C0" Offset="0x30C0" />
-        <Texture Name="object_ms_TLUT_003230" OutName="tlut_003230" Format="rgba16" Width="16" Height="16" Offset="0x3230" />
-        <Texture Name="object_ms_Tex_003430" OutName="tex_003430" Format="ci8" Width="16" Height="16" Offset="0x3430" />
-        <Texture Name="object_ms_Tex_003530" OutName="tex_003530" Format="ci8" Width="32" Height="32" Offset="0x3530" />
-        <Texture Name="object_ms_Tex_003930" OutName="tex_003930" Format="ci8" Width="16" Height="16" Offset="0x3930" />
-        <Texture Name="object_ms_Tex_003A30" OutName="tex_003A30" Format="ci8" Width="8" Height="8" Offset="0x3A30" />
-        <Texture Name="object_ms_Tex_003A70" OutName="tex_003A70" Format="ci8" Width="8" Height="8" Offset="0x3A70" />
-        <Texture Name="object_ms_Tex_003AB0" OutName="tex_003AB0" Format="ci8" Width="8" Height="8" Offset="0x3AB0" />
-        <Texture Name="object_ms_Tex_003AF0" OutName="tex_003AF0" Format="ci8" Width="16" Height="16" Offset="0x3AF0" />
-        <Texture Name="object_ms_Tex_003BF0" OutName="tex_003BF0" Format="ci8" Width="8" Height="8" Offset="0x3BF0" />
-        <Texture Name="object_ms_Tex_003C30" OutName="tex_003C30" Format="ci8" Width="4" Height="4" Offset="0x3C30" />
-        <Texture Name="object_ms_Tex_003C40" OutName="tex_003C40" Format="ci8" Width="16" Height="16" Offset="0x3C40" />
-        <Limb Name="object_ms_Standardlimb_003D40" Type="Standard" EnumName="OBJECT_MS_LIMB_01" Offset="0x3D40" />
-        <Limb Name="object_ms_Standardlimb_003D4C" Type="Standard" EnumName="OBJECT_MS_LIMB_02" Offset="0x3D4C" />
-        <Limb Name="object_ms_Standardlimb_003D58" Type="Standard" EnumName="OBJECT_MS_LIMB_03" Offset="0x3D58" />
-        <Limb Name="object_ms_Standardlimb_003D64" Type="Standard" EnumName="OBJECT_MS_LIMB_04" Offset="0x3D64" />
-        <Limb Name="object_ms_Standardlimb_003D70" Type="Standard" EnumName="OBJECT_MS_LIMB_05" Offset="0x3D70" />
-        <Limb Name="object_ms_Standardlimb_003D7C" Type="Standard" EnumName="OBJECT_MS_LIMB_06" Offset="0x3D7C" />
-        <Limb Name="object_ms_Standardlimb_003D88" Type="Standard" EnumName="OBJECT_MS_LIMB_07" Offset="0x3D88" />
-        <Limb Name="object_ms_Standardlimb_003D94" Type="Standard" EnumName="OBJECT_MS_LIMB_08" Offset="0x3D94" />
-        <Skeleton Name="object_ms_Skel_003DC0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_MS_LIMB_NONE" LimbMax="OBJECT_MS_LIMB_MAX" EnumName="ObjectMsLimb" Offset="0x3DC0" />
```

Comment:
You accidentally threw away some useful information regarding limb naming, that shouldn't be removed but expanded to match the rest of the object, so that future documentation of beanguy can use it.